src/Magento/FunctionalTestingFramework/Console/RunTestCommand.php

```diff
@@ -90,21 +90,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $returnCode = 0;
+        $codeceptionCommand = realpath(PROJECT_ROOT . '/vendor/bin/codecept') . ' run functional ';
+        $testsDirectory = TESTS_MODULE_PATH . DIRECTORY_SEPARATOR . TestGenerator::GENERATED_DIR . DIRECTORY_SEPARATOR;
         //execute only tests specified as arguments in run command
         foreach ($tests as $test) {
-            $codeceptionCommand = realpath(PROJECT_ROOT . '/vendor/bin/codecept') . ' run functional ';
-            $test = TESTS_MODULE_PATH .
-                DIRECTORY_SEPARATOR .
-                TestGenerator::GENERATED_DIR .
-                DIRECTORY_SEPARATOR .
-                TestGenerator::DEFAULT_DIR .
-                DIRECTORY_SEPARATOR .
-                $test .
-                'Cest.php';
-            $codeceptionCommand .= $test;
-            $codeceptionCommand .= ' --verbose --steps';
-
-            $process = new Process($codeceptionCommand);
+            $testGroup = TestGenerator::DEFAULT_DIR . DIRECTORY_SEPARATOR;
+            $testName = $test . 'Cest.php';
+            $fullCommand = $codeceptionCommand . $testsDirectory . $testGroup . $testName . ' --verbose --steps';
```

Comment:
How do you feel about adding in some explicit error checking:
```
if (!realpath($testsDirectory . $testGroup . $testName))
```

Right now if I try to reference a test that isn't generated I get a massive stack trace and an exception in Loader.php